npm.mk

```diff
@@ -0,0 +1,23 @@
+npm = docker run -it --rm \
```

Comment:
J'aurais tendance à dire que c'est un problème à régler côté CI. Puis docker dans docker çà n'est pas conseillé mais ca foncitonne et il y a docker dans LXC/LXD qui fonctionne très bien.
Je connais mal les CI mais je suis prêt à parier qu'ils gèrent çà. [Docker in travis](https://docs.travis-ci.com/user/docker/)